### PR TITLE
Extra documentation for Schannel ciphers

### PR DESCRIPTION
**Pull request by georgeok**

### CODE REVIEW

File: docs/CIPHERS.md [hunk 1]
```diff
@@ -461,6 +461,14 @@ specific ciphersuites. They are
 [defined](https://docs.microsoft.com/windows/desktop/SecCrypto/alg-id) by
 Microsoft.
 
+There is also the case that the selected algorithm will not match the ciphers
+offered by the server during the SSL negotiation. In this case curl will 
+print an `CURLE_SSL_CONNECT_ERROR (35 SEC_E_ALGORITHM_MISMATCH` and the
+request will fail.
+
+If no algorithm is specified, curl (and subsequently Schannel) will use the
+ciphers defined in the registry `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\ciphers`.
```
Comment (jay) [inline docs/CIPHERS.md]: Where is this documented? I can't find that this is true.

File: docs/CIPHERS.md [hunk 2]
```diff
@@ -461,6 +461,14 @@ specific ciphersuites. They are
 [defined](https://docs.microsoft.com/windows/desktop/SecCrypto/alg-id) by
 Microsoft.
 
+There is also the case that the selected algorithm will not match the ciphers
+offered by the server during the SSL negotiation. In this case curl will 
+print an `CURLE_SSL_CONNECT_ERROR (35 SEC_E_ALGORITHM_MISMATCH` and the
```
Comment (jay) [inline docs/CIPHERS.md]: I think you meant (35) SEC_E_ALGORITHM_MISMATCH?

File: docs/CIPHERS.md (hunk 1 above)
Comment (georgeok) [reply docs/CIPHERS.md]: I found a few information in https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/manage-ssl-protocols-in-ad-fs#enable-and-disable-tls-10. There is another registry path probably worth mentioning `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Cryptography\Configuration\Local\SSL\00010002` https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/manage-ssl-protocols-in-ad-fs#enabling-or-disabling-additional-cipher-suites

File: docs/CIPHERS.md (hunk 2 above)
Comment (georgeok) [reply docs/CIPHERS.md]: Yes.. Doing a force push to update it

File: docs/CIPHERS.md (hunk 1 above)
Comment (jay) [reply docs/CIPHERS.md]: I think this paragraph should be left out. I am using a Windows 7 computer right now that does not have anything set for that key. It's implied that if no cipher is set curl will use the default ciphers.

**Comment by jay** [comment]
Thanks

**Pull request closed without merging**